addons/edi/models/edi_document.py

```diff
@@ -170,6 +176,16 @@ def autoemit(self):
             docs += doc
         return docs
 
+    @api.onchange("fail_fast", "fail_end")
+    def onchange_fail_configurations(self):
```

Comment:
worth adding a constraint for this too, else we can likely get the bad configuration through data xml

Reply:
Data xml is controlled by us, and the CR-s are reviewed normally both of them shouldnt be check, but even in case they are for some reason it will act as fail_fast. I mean can add, just think is not needed.

Reply:
i'd rather have it. i'm not going to remember to check on every CR that both should not be set